deltachat-ios/Controller/AddGroupMembersViewController.swift

```diff
@@ -153,20 +154,6 @@ class AddGroupMembersViewController: GroupMembersViewController {
         navigationController?.popViewController(animated: true)
     }
 
-    func getNewContactCell() -> UITableViewCell {
```

Comment:
method removed in favor of a lazy variable containing an ActionCell, streamlining to have more consistency with other static cells.